ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/LedgerDao.scala

```diff
@@ -133,13 +131,13 @@ trait LedgerReadDao extends ReportsHealth {
   def getLfArchive(packageId: PackageId): Future[Option[Archive]]
 
   /** Returns a stream of package upload entries.
-    * @param startInclusive starting offset inclusive
-    * @param endExclusive   ending offset exclusive
+    * @param startExclusive starting offset inclusive
+    * @param endInclusive   ending offset exclusive
```

Comment:
Either fix the comments or delete them entirely, they don't add any information.

Reply:
Will delete. See 949dc9225 in #4953.